ci3/test/avm-transpiler

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I kinda think its weird we're putting project specifics in the ci3 folder.
Shouldn't the ci3 scripts be "agnostic" to what they're building?